src/main/java/com/amazon/ion/system/IonSystemBuilder.java

```diff
@@ -271,7 +276,111 @@ public final IonSystemBuilder withStreamCopyOptimized(boolean optimized)
         return b;
     }
 
+    //=========================================================================
+
+    /**
+     * Gets the text writer builder whose options will be used when building an
+     * {@link IonSystem}. By default, {@link IonTextWriterBuilder#standard()} will
+     * be used.
+     *
+     * @see #setIonTextWriterBuilder(IonTextWriterBuilder)
+     * @see #withIonTextWriterBuilder(IonTextWriterBuilder)
+     */
+    public final IonTextWriterBuilder getIonTextWriterBuilder() {
+        return textWriterBuilder;
+    }
+
+    /**
+     * Sets the text writer builder whose options will be used to use when building
+     * an {@link IonSystem}. The writer builder's catalog will never be used; the
+     * catalog provided to {@link #setCatalog(IonCatalog)} or
+     * {@link #withCatalog(IonCatalog)} will always be used instead.
+     *
+     * @param builder the writer builder to use in built systems.
+     *  If null, each system will be built with {@link IonTextWriterBuilder#standard()}.
```

Comment:
This says that the default is `IonTextWriterBuilder.standard()`, but in the `build()` method, the default is actually `IonTextWriterBuilder.standard().withCharsetAscii()`. Can you make sure that this javadoc and the javadoc for `withIonTextWriterBuilder()` mention `withCharsetAscii()`?

The default charset for a text writer builder is actually `UTF-8`, so if a user tried to e.g. configure the newline type like this: 
```java
IonTextWriterBuilder twb = IonTextWriterBuilder.standard().withNewLineType(NewLineType.LF);
IonSystem ionSystem = IonSystemBuilder.standard()
    .withIonTextWriterBuilder(twb)
    .build();
```
In addition to the intended change to the new line configuration, they would unexpectedly change the default charset from ASCII to UTF-8.

Reply:
Wouldn't it be a good idea to harmonize the default values? Using UTF-8 on the one hand and ASCII on the other is indeed confusing.

Ideally, builders should manage their own default values.

I added the default encoding in the javadoc of the different methods related to this builder.

Reply:
It would be good to harmonize the default values. However, we need to be careful of any potentially breaking changes.